src/components/node-list/node-list-group.js

```diff
@@ -35,31 +35,29 @@ export const NodeListGroup = ({
       )}
     >
       <h3 className="pipeline-nodelist__heading">
-        <NodeListRow
+        <FilterRow
```

Comment:
I am not sure if we need all ul, li code now ? it works without it because of FilterRow

Reply:
hey @rashidakanchwala thanks for spotting this, as discussed I've removed all the un-used `li` elements, and add an extra props to `FilterRow` to determine whether it should be a div or `li` element. The HTML structure will now look something like in the screenshot here

<img width="723" alt="Screenshot 2024-10-28 at 16 46 33" src="https://github.com/user-attachments/assets/0cd34cf0-4a6f-45b8-983f-1781c84c78ee">